src/components/carousel/carousel.stories.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, boolean } from '@storybook/addon-knobs';
+import notes from './notes.md';
+import { Carousel, Slide } from './carousel.js';
+
+storiesOf('Carousel', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [Slide]
+    }
+  })
+  .add('default', () => {
+    const initialSlideIndex = select('initialSlideIndex', [0, 1, 2, 3, 4], Carousel.defaultProps.initialSlideIndex);
+    const slideIndex = select('slideIndex', [0, 1, 2, 3, 4], 0);
+    const enableSlideSelector = boolean('enableSlideSelector', Carousel.defaultProps.enableSlideSelector);
+    const enablePreviousButton = boolean('enablePreviousButton', Carousel.defaultProps.enablePreviousButton);
+    const enableNextButton = boolean('enableNextButton', Carousel.defaultProps.enableNextButton);
+    const transition = select('transition', ['slide', 'fade'], Carousel.defaultProps.transition);
```

Comment:
Some new lines in the above variable definitions might make this a bit more readable.

Reply:
Do you mean brake lines?

Reply:
I think something like this:

```
const initialSlideIndex = select(
  'initialSlideIndex',
  [0, 1, 2, 3, 4],
  Carousel.defaultProps.initialSlideIndex
);
```

Reply:
sure thanks